tarantool_test.go

```diff
@@ -1308,6 +1429,113 @@ func TestStressSQL(t *testing.T) {
 	}
 }
 
+func TestNewPrepared(t *testing.T) {
```

Comment:
To make it easier to understand you can split the test into 2:
1. Connection.NewPrepared()
2. NewPreparedFromResponse()

Reply:
Splitted.

Reply:
```Go
	prepareReq := NewPrepareRequest(selectNamedQuery2)
	resp, err = conn.Do(prepareReq).Get()
	if err != nil {
		t.Errorf("failed to prepare: %v", err)
	}
	if resp.Code != OkCode {
		t.Errorf("failed to unprepare prepared statement: code %d", resp.Code)
	}

	stmt, err = NewPreparedFromResponse(conn, resp)
	if err != nil {
		t.Errorf("failed to construct a prepared object: %v", err)
	}
	if stmt.Conn != conn {
		t.Errorf("the statement does not belong to the passed connection")
	}
```

The code still present in `TestNewPrepared`.